[2023-07-26T15:47:02.454-0300] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: LH_ED_GABRIELLEMUNIZ.write_second_db backfill__2023-07-18T00:00:00-03:00 [queued]>
[2023-07-26T15:47:02.462-0300] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: LH_ED_GABRIELLEMUNIZ.write_second_db backfill__2023-07-18T00:00:00-03:00 [queued]>
[2023-07-26T15:47:02.462-0300] {taskinstance.py:1308} INFO - Starting attempt 1 of 4
[2023-07-26T15:47:02.482-0300] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): write_second_db> on 2023-07-18 03:00:00+00:00
[2023-07-26T15:47:02.487-0300] {standard_task_runner.py:57} INFO - Started process 29666 to run task
[2023-07-26T15:47:02.492-0300] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'LH_ED_GABRIELLEMUNIZ', 'write_second_db', 'backfill__2023-07-18T00:00:00-03:00', '--job-id', '427', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/LH_ED_GABRIELLEMUNIZ.py', '--cfg-path', '/tmp/tmpdngcavke']
[2023-07-26T15:47:02.493-0300] {standard_task_runner.py:85} INFO - Job 427: Subtask write_second_db
[2023-07-26T15:47:02.545-0300] {task_command.py:410} INFO - Running <TaskInstance: LH_ED_GABRIELLEMUNIZ.write_second_db backfill__2023-07-18T00:00:00-03:00 [running]> on host ubuntu
[2023-07-26T15:47:02.646-0300] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='gabi' AIRFLOW_CTX_DAG_ID='LH_ED_GABRIELLEMUNIZ' AIRFLOW_CTX_TASK_ID='write_second_db' AIRFLOW_CTX_EXECUTION_DATE='2023-07-18T03:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='backfill__2023-07-18T00:00:00-03:00'
[2023-07-26T15:47:02.685-0300] {logging_mixin.py:150} WARNING - /home/gabi/Documentos/Gabi/Indicium/code-challenge-indicium/ind_env/lib/python3.10/site-packages/airflow/utils/context.py:313 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2023-07-26T15:47:03.422-0300] {python.py:183} INFO - Done. Returned value was: None
[2023-07-26T15:47:03.428-0300] {taskinstance.py:1345} INFO - Marking task as SUCCESS. dag_id=LH_ED_GABRIELLEMUNIZ, task_id=write_second_db, execution_date=20230718T030000, start_date=20230726T184702, end_date=20230726T184703
[2023-07-26T15:47:03.466-0300] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-07-26T15:47:03.500-0300] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
